ceph-ansible-nightly/config/definitions/ceph-ansible-nightly.yml

```diff
@@ -16,8 +16,12 @@
       - purge_cluster_non_container
       - purge_filestore_osds_container
       - purge_filestore_osds_non_container
+    ceph_ansible_branch:
+      - stable-2.2
```

Comment:
please keep `v2.2.11` tag.

Reply:
to use a tag jenkins requires it to be in the format of ``tags/$tag``. In this case it'd be ``tags/v2.2.11``

Reply:
@leseb could you explain why you need to test a tag every night? A tag never changes, so the results would always be the same. If you want to verify that a given tag works, it could be done with a one-time run. There is such a job already in https://2.jenkins.ceph.com/job/ceph-ansible-scenario/

Reply:
I understand your concern, but what if I want to test all the scenarios against one tag?
How should I do that?

Reply:
@leseb The current nightly jobs also take a ceph-ansible branch or tag as a parameter. For example, I just started this nightly against ``tags/v2.2.11``: https://2.jenkins.ceph.com/job/ceph-ansible-nightly-jewel-ansible2.3-centos7_cluster/78/console

The job that @alfredodeza linked to also allows you to specify which scenario you'd like to test as well as the ceph-ansible and ceph branch.

Once this PR merges it'll just create new jobs, each defaulting to the branch or tag defined here.